smash/tests/core/simulation/test_bayesian_optimize.py

```diff
@@ -10,7 +10,7 @@
 def generic_custom_bayesian_optimize(model: smash.Model, **kwargs) -> dict:
     res = {}
 
-    ncpu = max(1, os.cpu_count() - 1)
+    ncpu = min(5, os.cpu_count())
```

Comment:
Je propose ça pour éviter le fait d'utiliser tous les cpus pour les machines ayant que 1-5 cpus:
```suggestion
    ncpu = min(5, max(1, os.cpu_count() - 1))
```